playbooks/roles/caasp4/tasks/_assert_packages.yml

```diff
@@ -0,0 +1,73 @@
+---
+#
+# (c) Copyright 2019 SUSE Linux GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+- name: Identify localhost SUSE OS distribution and version
+  set_fact:
+    suse_repo_version: "CAASP40-{{ansible_distribution}}-{{ansible_distribution_version | replace('.', '') }}"
+  when:
+    - ansible_distribution | lower == "sles"
+
+- name: Configure repositories for skuba and terraforms
+  become: yes
+  zypper_repository:
+    name: "{{ suse_repo_version }}"
+    repo: "{{ socok8s_repos_to_configure[suse_repo_version] }}"
+    autorefresh: True
+    auto_import_keys: yes
+    state: present
+  when:
+    - suse_repo_version is defined
+    - suse_repo_version in socok8s_repos_to_configure
+
+- name: Install required skuba, terraform packages for OS versions
+  become: yes
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - skuba
+    - terraform
```

Comment:
dont we need to lock terraform to 0.11 here? In case the ansible runner has another repo which contains a higher version of terraform

Reply:
That check is done later. 
ansible package module (https://docs.ansible.com/ansible/2.5/modules/package_module.html) does not provide a mechanism to install specific version for suse.

I am assuming down the line, we may not have that requirement as repository configured will have working compatible version

Reply:
package nope, but the zypper module does https://docs.ansible.com/ansible/latest/modules/zypper_module.html

Reply:
Yes, so do we need specific sub-version e.g. right now I see v0.11.11 ? I mean will v0.11.10 work ? Asking as I existing version check is done only for v0.11.xx .

I don't see if there is way to restrict sub-version in that format 
`# Install specific version (possible comparisons: <, >, <=, >=, =)
- zypper:
    name: 'docker>=1.10'
    state: present`

Reply:
Any parameter of the zypper command can also be used here.

Not sure it's worth freezing this. @kantarun is good enough, we shouldn't spend too much time on this.

Reply:
0.11.x should be ok